routing/routing_settings.cpp

```diff
@@ -23,24 +24,32 @@ RoutingSettings GetRoutingSettings(VehicleType vehicleType)
   {
   case VehicleType::Pedestrian:
     return {false /* useDirectionForRouteBuilding */,
-            true /* m_matchRoute */,         false /* m_soundDirection */,
-            20. /* m_matchingThresholdM */,  true /* m_keepPedestrianInfo */,
-            false /* m_showTurnAfterNext */, -1 /* m_minSpeedForRouteRebuildMpS */};
+            false /* m_matchRoute */,
+            false /* m_soundDirection */,
+            20. /* m_matchingThresholdM */,
+            false /* m_showTurnAfterNext */,
+            -1 /* m_minSpeedForRouteRebuildMpS */};
   case VehicleType::Transit:
     return {false /* useDirectionForRouteBuilding */,
-            true /* m_matchRoute */,         false /* m_soundDirection */,
-            40. /* m_matchingThresholdM */,  true /* m_keepPedestrianInfo */,
-            false /* m_showTurnAfterNext */, -1 /* m_minSpeedForRouteRebuildMpS */};
+            true /* m_matchRoute */,
+            false /* m_soundDirection */,
+            40. /* m_matchingThresholdM */,
+            false /* m_showTurnAfterNext */,
+            -1 /* m_minSpeedForRouteRebuildMpS */};
   case VehicleType::Bicycle:
     return {false /* useDirectionForRouteBuilding */,
-            true /* m_matchRoute */,         true /* m_soundDirection */,
-            30. /* m_matchingThresholdM */,  false /* m_keepPedestrianInfo */,
-            false /* m_showTurnAfterNext */, -1 /* m_minSpeedForRouteRebuildMpS */};
+            true /* m_matchRoute */,
+            true /* m_soundDirection */,
+            30. /* m_matchingThresholdM */,
+            false /* m_showTurnAfterNext */,
+            -1 /* m_minSpeedForRouteRebuildMpS */};
   case VehicleType::Car:
     return {true /* useDirectionForRouteBuilding */,
-            true /* m_matchRoute */,        true /* m_soundDirection */,
-            50. /* m_matchingThresholdM */, false /* m_keepPedestrianInfo */,
-            true /* m_showTurnAfterNext */, routing::KMPH2MPS(3) /* m_minSpeedForRouteRebuildMpS */};
+            true /* m_matchRoute */,
+            true /* m_soundDirection */,
+            50. /* m_matchingThresholdM */,
+            true /* m_showTurnAfterNext */,
+            routing::KMPH2MPS(3) /* m_minSpeedForRouteRebuildMpS */};
```

Comment:
put `.0` to double number at this file please

Reply:
Done